plugins/lando/lando.plugin.zsh

```diff
@@ -0,0 +1,54 @@
+LANDO_ZSH_SITES_DIRECTORY="$HOME/Sites"
+
+LANDO_ZSH_CONFIG_FILE=./.lando.yml
+
+# Enable multiple commands with lando.
+function  artisan \
+          composer \
+          drush \
+          gulp \
+          npm \
+          wp \
+          yarn {
+
+  if checkForFile ; then
+    lando "$0" "$@"
+  else
+    command "$0" "$@"
+  fi
+}
+
+# Check for the file in the current and parent directories.
+checkForFile(){
+
+  local current_directory="$PWD"
+
+  # Bash is backwards. 0 is true 1 (non-zero) is false.
+  # Only bother checking for lando within the Sites directory.
+  if [[ "$PWD" == "$LANDO_ZSH_SITES_DIRECTORY"* ]]; then
+
+    # "Checking for file: $LANDO_ZSH_CONFIG_FILE within $LANDO_ZSH_SITES_DIRECTORY..."
+
+    while true; do
+      if [ $current_directory != "$LANDO_ZSH_SITES_DIRECTORY" ]; then
+          if [ -f "$current_directory/$LANDO_ZSH_CONFIG_FILE" ]; then
+            echo "Using Lando"
+            return
+          fi
+        current_directory="$(dirname $current_directory)"
```

Comment:
Instead of dirname you can use `${current_directory:h}` which gets the dirname (zsh specific).